protected/humhub/modules/admin/controllers/InformationController.php

```diff
@@ -8,24 +8,30 @@
 
 namespace humhub\modules\admin\controllers;
 
+use humhub\libs\SelfTest;
 use humhub\modules\admin\components\Controller;
 use humhub\modules\admin\components\DatabaseInfo;
 use humhub\modules\admin\libs\HumHubAPI;
+use humhub\modules\admin\permissions\SeeAdminInformation;
 use humhub\modules\queue\driver\MySQL;
 use humhub\modules\queue\helpers\QueueHelper;
 use humhub\modules\queue\interfaces\QueueInfoInterface;
 use humhub\modules\search\jobs\RebuildIndex;
+use humhub\services\MigrationService;
 use ReflectionClass;
 use ReflectionException;
 use Yii;
 
 /**
- * Informations
+ * Information
  *
  * @since 0.5
  */
 class InformationController extends Controller
```

Comment:
Related to another PR?

Reply:
I leave this up to you. I honestly don't have the time to create an extra pr for this. I'll remove the change here.